projects/plugins/jetpack/extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -61,8 +61,8 @@ public static function render( $attr, $content ) {
 
 		$is_squareish_layout = self::is_squareish_layout( $attr );
 
-		// Jetpack_Plan does not exist on WordPress.com.
-		if ( class_exists( 'Jetpack_Plan' ) ) {
+		// Jetpack_Plan::get does not exist on WordPress.com.
+		if ( method_exists( 'Jetpack_Plan', 'get' ) ) {
```

Comment:
This prevents fatals on WP.com since we're adding a `Jetpack_Plan` class there with only the `supports` method (see D75906-code).

Reply:
Would it be better to fully sync Jetpack_Plan to wpcom instead, so when p9dueE-4mF-p2 and later work happens we don't have to figure out how to fix things then?

Reply:
I would this is okay and something we can fix properly when Jetpack is on WP.com.

Reply:
> fully sync Jetpack_Plan to wpcom instead

That'd require to refactor the `Jetpack_Plan` class in order to ensure its methods also work in WP.com. I took a briefly look at what would be needed, and it didn't seem trivial, so I decided to keep things simple for this PR.